hack/jenkins/e2e.sh

```diff
@@ -219,7 +224,10 @@ fi
 # Jenkins will look at the junit*.xml files for test failures, so don't exit
 # with a nonzero error code if it was only tests that failed.
 if [[ "${E2E_TEST,,}" == "true" ]]; then
-    go run ./hack/e2e.go ${E2E_OPT} -v --test --test_args="${GINKGO_TEST_ARGS}--ginkgo.noColor" || true
+    # GINKGO_ARGS is parsed by the Ginkgo runner itself, whereas the --test_args
```

Comment:
Except that it's not really true?

hack/e2e.go: 
testArgs         = flag.String("test_args", "", "Space-separated list of arguments to pass to Ginkgo test runner.")